tests/eas/acceptance.py

```diff
@@ -393,34 +393,45 @@ class WakeMigration(EasTest):
     """
 
     conf_basename = "wake_migration.config"
-    phase_duration = WORKLOAD_DURATION_S
 
     @experiment_test
     def test_first_cpu(self, experiment, tasks):
         """Wake Migration: Test First CPU"""
         self._do_test_first_cpu(experiment, tasks)
 
-    @experiment_test
-    def test_little_big_switch1(self, experiment, tasks):
-        """Wake Migration: LITTLE -> BIG: 1"""
-        expected_time = self.get_start_time(experiment) + self.phase_duration
-        switch_window = (
-            expected_time -
-            SWITCH_WINDOW_HALF,
-            expected_time +
-            SWITCH_WINDOW_HALF)
+    def _assert_switch(self, experiment, expected_switch_to, phases):
```

Comment:
Not completely sure... but perhaps which can be another candidate method to move into the base class?
I guess it can be quite usual to check for tasks switches in many different other test scenarios...